Casks/z/zen-browser@twilight.rb

```diff
@@ -0,0 +1,32 @@
+cask "zen-browser@twilight" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "1.0.1-t.20"
+  sha256 arm:   "67c560a1568ec084b4a0b7c82787396f77da12954a77a9e194e084555c757ae9",
+         intel: "45f643373f3acb6a275f91718d0821f56001cbe21449822df2dc0ac4d9f33620"
```

Comment:
```suggestion
  sha256 :no_check
```

We should use `:no_check` here because the `url` is unversioned.